src/main/java/pl/uj/io/cuteanimals/plot/actions/DungeonInvestigateAction.java

```diff
@@ -26,7 +26,7 @@ public IResult actionBody(IPlayer player) {
         }
 
         for (IItem item : player.getArmor().getItems()) {
-            if (item.getName().equals("torch")) {
+            if (item.getName().equals("Torch")) {
```

Comment:
```suggestion
            if (item.getName().toLower().equals("torch")) {
```
This should fix the issue